tuned/exports/dbus_exporter.py

```diff
@@ -59,15 +68,23 @@ def object_name(self):
 	def running(self):
 		return self._thread is not None
 
+	def _prepare_for_dbus(self, method, wrapper):
+		source = """def {name}({args}):
+					return wrapper({args})
+		""".format(name=method.__name__, args=', '.join(getargspec(method.__func__)))
+		code = compile(source, '<decorator-gen-%d>' % len(self._dbus_methods), 'exec')
+		func = FunctionType(code.co_consts[0], locals(), method.__name__)
```

Comment:
This needs comment, especially for the co_consts[0], preferably with the link to the documentation.